src/Raven.Client/Documents/Indexes/ReduceRunDetails.cs

```diff
@@ -1,6 +1,23 @@
 ﻿namespace Raven.Client.Documents.Indexes
 {
     public class ReduceRunDetails
+    {
```

Comment:
How about adding the amount of allocated memory in this thread?

Reply:
What kind of help would these details give us?

Reply:
1) Allocated memory in this thread is already here, it's `CurrentlyAllocated` property
2) It's the same kind of help like in map phase - those are exactly the same values we have there, so we know memory stats and details about numbers of processed map entries